### ISSUE TITLE
Fails on FIPS systems - bundled crypto module missing fix from Ansible 2.9

### ISSUE THREAD

**Issue by jeffcpullen** [kube-reporting/metering-operator#1444]
The operator fails to deploy on FIPS enabled systems with the following error

```
Traceback (most recent call last):
  File "/tmp/.ansible-/tmp/ansible-tmp-1605639519.1729925-24168-234385709602156/AnsiballZ_openssl_privatekey.py", line 102, in <module>
    _ansiballz_main()
  File "/tmp/.ansible-/tmp/ansible-tmp-1605639519.1729925-24168-234385709602156/AnsiballZ_openssl_privatekey.py", line 94, in _ansiballz_main
    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)
  File "/tmp/.ansible-/tmp/ansible-tmp-1605639519.1729925-24168-234385709602156/AnsiballZ_openssl_privatekey.py", line 40, in invoke_module
    runpy.run_module(mod_name='ansible.modules.crypto.openssl_privatekey', init_globals=None, run_name='__main__', alter_sys=True)
  File "/usr/lib64/python3.6/runpy.py", line 205, in run_module
    return _run_module_code(code, init_globals, run_name, mod_spec)
  File "/usr/lib64/python3.6/runpy.py", line 96, in _run_module_code
    mod_name, mod_spec, pkg_name, script_name)
  File "/usr/lib64/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmp/ansible_openssl_privatekey_payload_rg2zb9st/ansible_openssl_privatekey_payload.zip/ansible/modules/crypto/openssl_privatekey.py", line 692, in <module>
  File "/tmp/ansible_openssl_privatekey_payload_rg2zb9st/ansible_openssl_privatekey_payload.zip/ansible/modules/crypto/openssl_privatekey.py", line 676, in main
  File "/tmp/ansible_openssl_privatekey_payload_rg2zb9st/ansible_openssl_privatekey_payload.zip/ansible/modules/crypto/openssl_privatekey.py", line 303, in generate
  File "/tmp/ansible_openssl_privatekey_payload_rg2zb9st/ansible_openssl_privatekey_payload.zip/ansible/modules/crypto/openssl_privatekey.py", line 545, in _get_fingerprint
  File "/tmp/ansible_openssl_privatekey_payload_rg2zb9st/ansible_openssl_privatekey_payload.zip/ansible/module_utils/crypto.py", line 164, in get_fingerprint_of_bytes
ValueError: [digital envelope routines: EVP_DigestInit_ex] disabled for FIPS
```

I thought initially this was an issue with the community crypto collection as discussed [here](https://github.com/ansible-collections/community.crypto/issues/144) but I was pointed back to where this was an Ansible 2.9 issue which was resolved back in March with these fixes [openssl_* modules: prevent crash on fingerprint determination in FIPS mode #67515](https://github.com/ansible/ansible/pull/67515) and [[2.9] openssl_* modules: prevent crash on fingerprint determination in FIPS mode #67519](https://github.com/ansible/ansible/pull/67519)

This operator bundles a crypto.py that does not include the FIPS fix based on comparisons of the code. The relevant sections missing appear to be this:

crypto.py (lines starting around 158(ish)
```
        try:
            h = f(source)
        except ValueError:
            # This can happen for hash algorithms not supported in FIPS mode
            # (https://github.com/ansible/ansible/issues/67213)
            continue
```

**Comment by jeffcpullen** [comment]
Based on the conversations around this issue at https://github.com/ansible-collections/community.crypto/issues/144 the suggested fix here was just requiring Ansible >=2.9.6 and removing this forked crypto version.

**Comment by timflannagan1** [comment]
Looks like automation didn't close this out after the PR linked merged. Thanks for the BZ and for outlining the fix.

/close
